libs/chlorophyll/scss/components/chips/_index.scss

```diff
@@ -0,0 +1,34 @@
+@use '../../tokens/shame' as tokens;
+@use 'mixins';
+
+:root {
+  @include mixins.global-vars;
```

Comment:
These variables should be moved to the following places I think to work with global import and the .use-green mixin:
[Global scope](https://github.com/sebgroup/green/blob/main/libs/chlorophyll/scss/components/themes/_index.scss#L81)
[use green scope](https://github.com/sebgroup/green/blob/main/libs/chlorophyll/scss/components/use-green-scope/_index.scss#L17)